ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -96,13 +95,9 @@ private static bool SceneObjectShouldConvertToHammerMesh(string modelName)
     /// <summary>
     /// Extract a map from a resource. Accepted types include Map, World. TODO: WorldNode and EntityLump.
     /// </summary>
-    public MapExtract(Resource resource, IFileLoader fileLoader, object fileFlags = null)
+    public MapExtract(Resource resource, IFileLoader fileLoader, ResourceOptions fileFlags)
```

Comment:
I would have it take VmapOptions

Reply:
ah my bad, i thought i did alreayd, might have accidentally undid it